### ISSUE TITLE
Providing FFT summary info by `summary.FFTrees(x)`

### ISSUE THREAD

**Issue by hneth** [ndphillips/FFTrees#75]
Due to some earlier revision of the `FFTrees` object structure, the function call `summary.FFTrees(x)` no longer yields any useful outputs.

This bug can easily be fixed, of course, but reminds me of a question I was already wondering about when revising the `print()` and `plot()` methods: 

- Is there some deep reason for re-computing many measures from basic frequencies (`hi`, `mi`, `fa` and `cr`) when printing or plotting trees? 

In other words:

- Can we generally rely on the information in an `FFTrees` object `x` (e.g., the performance details stored in `x$trees$stats`) OR should we rather mistrust this data and re-compute measures from basic frequencies when possible? 

**Suggestion**

Unless there is reason to mistrust the object data, I'd simply retrieve and report the stats from `x$trees$stats`.

**Comment by hneth** [comment]
PS: This issue is fixed by PR #76.

**Comment by hneth** [comment]
PR #76 has just been merged.
